### PR TITLE
general cleanup based on review

### PR DESCRIPTION
**Pull request by xavluiz**
basic cleanup

### CODE REVIEW

File: src/managers/TrackerManager.ts [hunk 1]
```diff
@@ -107,7 +107,7 @@ export class TrackerManager {
         ...repoParams,
       };
 
-      const resp = await swdcTracker.trackCodeTimeEvent(codetime_event);
+      await swdcTracker.trackCodeTimeEvent(codetime_event);
```
Comment (mmclead) [inline src/managers/TrackerManager.ts:110]: Do we need the await at all here?   Or can we let this just happen on the event loop

Comment (xavluiz) [reply src/managers/TrackerManager.ts:110]: we can just let it happen. will remove that as well

Comment (xavluiz) [reply src/managers/TrackerManager.ts]: updated

Comment (mmclead) [reply src/managers/TrackerManager.ts]: cool this works well.

**Pull request merged**